### ISSUE TITLE
Playlists added in another Spotify app are not picked up

### ISSUE THREAD

**Issue by MartinWallgren** [xou816/spot#94]
**Describe the bug**
Playlists added in another spotify app are not picked up, not even after a restart of Spot.

**To Reproduce**
1. Go to 'the Playlist view in Spot'
2. Open a playlist that you are not yet following in another spotify app (for example web, mobile or anyone)
3. Follow the playlist
4. The playlists view in Spot is never updated.

**Expected behavior**
Expected the new playlist to appear in my Playlists view.

**Screenshots**
N/A

**General information:**
 - Distribution: Debian
 - Installation method [e. g. built from source, installed from Flathub...]: Flathub
 - Version [e.g. 0.1.0]:  0.1.8
 - Device used [e. g. desktop, phone...]: Desktop

**Stack trace:**
```
N/A
```

**Additional context**
I understand that Spot is in early development, but I think some mechanism to react to changes made outside Spot to the spotify account is needed at some point.

**Comment by xou816** [comment]
Hi! Thanks for the report.

Playlists are picked up, but not immediately: the cache is not very smart and could use some improvments!

**Comment by MartinWallgren** [comment]
Indeed, today the new playlists where there. Thank you for the reply.

**Comment by xou816** [comment]
No problem! I'm keeping this open to remind myself to look into proper cache management
